make-util.js

```diff
@@ -331,6 +331,9 @@ exports.ensureTool = ensureTool;
 
 var installNode = function (nodeVersion) {
```

Comment:
@merlynomsft InstallNode function is determining platform and then installing node from nodejs.org. Maybe we can use this function instead of introducing nvm?

Reply:
Please investigate and experiment.   My 2 cents: I believe nvm is a nice solution between it switches between already installed versions of node